src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -149,9 +160,9 @@ private CookieOptions BuildCookieOptions()
 
             var cookieValue = Options.TicketDataFormat.Protect(model);
 
-            if (model.Properties.IsPersistent)
+            if (model.Properties.IsPersistent && _renewExpiresUtc.HasValue)
             {
-                cookieOptions.Expires = _renewExpiresUtc.ToUniversalTime().DateTime;
+                cookieOptions.Expires = _renewExpiresUtc.Value.ToUniversalTime().DateTime;
```

Comment:
Why call ToUniversalTime? the name of the variable implies UTC, if it is not, that is an error.